docs/UserGuide.md

```diff
@@ -258,34 +258,39 @@ It consists of three main parts:
 * [Code Panel](#code-panel)
 
 ### Tool Bar
-The `Tool Bar` at the top provides a set of filters that control the chart panel. From right to left, the filters are:
-- Sort : Users can sort by:
-    - Total Contribution : The amount of lines, written by the author, in the repository.
-    - Variance : The variance of contribution of all commits.
-        - It indicates the code consistency of an author.
-        - It helps to determine whether the author has been contributing regularly or procrastinating.
-        - This can be useful for instructors of student projects, e.g. [Addressbook](https://github.com/se-edu/addressbook-level4).
-    - Author Name : The Author's GitHub ID or Display Name configured in the `CSV Config File`.
-    - Team Name : The name of the organization of the repository.
-- Interval : Interval refers to amount of time one single ramp represents.
-    - Users can choose two modes of time intervals:
-        - daily
-        - weekly
-- Period : The time period that the ramp charts display.
-- Group : Checkbox.
-    - If checked, authors that have contributed to the same repository will be displayed next to each other.
-    - This overrides the sort function.
-- Search : Filters the author and repository by keywords.
-    - Multiple keywords can be used, separated by spaces.
-    - The keywords are logically connected with OR operators.
-- Bookmark : The star icon at the top right corner.
-    - Clicking on it will generates a link to the report with all the tooltip settings.
-    - This link will be copied to user’s clipboard.
+The `Tool Bar` at the top provides a set of filters that control the chart panel.
+- Search : filters the author and repository by keywords.
+  - Multiple keywords can be used, separated by spaces.
+  - The keywords are logically connected with OR operators.
+- Sort by :
+  - Total Contribution : the amount of lines, written by the author, in the repository.
+  - Variance : the variance of contribution of all commits.
+    - It indicates the code consistency of an author.     
+    - It helps to determine whether the author has been contributing regularly or procrastinating.
```

Comment:
best not to mention things like this ` or procrastinating`. it is up to the user to make this kind of inferences.